corehq/apps/reports/standard/cases/case_data.py

```diff
@@ -522,7 +522,7 @@ def resave_case_view(request, domain, case_id):
     resave_case(domain, case)
     messages.success(
         request,
-        _('Case %s was successfully saved. Hopefully it will show up in all reports momentarily.' % case.name),
+        _('Case %s was successfully saved. Please allow a few minutes for the change to be reflected in all reports.' % case.name),  # noqa: E501
```

Comment:
```suggestion
        _('Case {} was successfully saved. Please allow a few minutes for the change to be reflected in all reports.').format(case.name),  # noqa: E501
```
As-is, the string interpolation happens inside the `_()`, so the result isn't actually translatable.